lib/pg_audit_log/extensions/postgresql_adapter.rb

```diff
@@ -30,29 +30,39 @@ def rename_table(table_name, new_name)
     end
   end
 
-  def reconnect!
+  def reconnect!(...)
     super
     @last_user_id = @last_unique_name = nil
   end
 
-  def execute(sql, name = nil)
+  def execute(...)
     set_audit_user_id_and_name
-    super(sql, name = nil)
+    super
+  end
+
+  def exec_query(...)
+    set_audit_user_id_and_name
+    super
+  end
+
+  def exec_insert(...)
+    set_audit_user_id_and_name
+    super
   end
 
-  def exec_query(*args, **kwargs, &block)
+  def exec_insert_all(...)
```

Comment:
This also calls `internal_exec_query` as of Rails 7.1: https://github.com/rails/rails/blob/v7.1.5.1/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb#L170